### DISCUSSION TITLE
能否自动调用motrix？

### DISCUSSION THREAD

**Discussion by sky-279** [the1812/Bilibili-Evolved#3300] [提问 / Q&A] [answered]
在下载视频的时候总是需要先手动打开motrix，否则会下载失败。能否做到类似于百度网盘那种浏览器自动打开应用的效果？

**Comment by theblueandblackcrow** [comment]
能啊
我用的就是motrix下载视频
要想调用的话，需要把端口这些信息填好
（话说这么冷门的软件还有人用我是真的没有想到😅）

**Reply by sky-279** [reply]
> 
额 我的意思是motrix并不是开机自启动的，如何在浏览器下载视频的时候自动启动motrix，motrix没在托盘的时候不是下载会报错嘛

**Reply by theblueandblackcrow** [reply]
那你这种功能恐怕不能通过motrix实现
我是设置了自启动的motrix
这样子就和你说的就不符了

**Reply by FoundTheWOUT** [reply]
我觉得还挺好用的（

**Reply by theblueandblackcrow** [reply]
> 我觉得还挺好用的（

motrix下阿b视频速度确实还可以

**Comment by FoundTheWOUT** [comment, marked as answer]
试用插件 [https://raw.githubusercontent.com/FoundTheWOUT/Bilibili-Evolved/master/registry/dist/plugins/video/download/motrix-output.js](https://raw.githubusercontent.com/FoundTheWOUT/Bilibili-Evolved/master/registry/dist/plugins/video/download/motrix-output.js)
### 限制
1. 貌似不能下载两个以上任务，估计可能query不能太长
2. 下载任务名应该是不能随便改，这和motrix设计有关。~~要改的话估计要给motrix提PR~~
